.github/workflows/anacopy.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        py: [3.7, 3.8, 3.9, "3.10"]
+        py: [3.7, 3.8, 3.9, "3.10", "3.11"]
```

Comment:
I think the reason `"3.10"` is in quotes is because YAML was dropping the 0 and converting it to `3.1`. The quotes shouldn't be needed for `3.11` in either of these files, but I'd be curious to hear if you've found otherwise!

Reply:
You're right. I changed it in a new commit. You can squash to clean it up, if you like.